Lushu/src/main/kotlin/lushu/ContextGrammar/Grammar/Rules.kt

```diff
@@ -0,0 +1,210 @@
+package lushu.ContextGrammar.Grammar
+
+class Rules(private val root: Node = Node()) {
+
+    var line: MutableList<String> = mutableListOf()
+    var auxiliarLine: MutableList<String> = mutableListOf()
+    private fun isEndingPlusMatcher(current: Node?, index: Int): Boolean{
+
+        if(line[index].endsWith(logSeparator))
+            return true
+        else if(current == null){
+            return false
+        }
+        for(child in current!!.getChildren()){
+            if(child.match(line[index])){
+                if(child.isSensitive())
+                    auxiliarLine[index] = "*".repeat(auxiliarLine[index].length)
+                return true
+            }
+
+        }
+
+        return false
+    }
+    data class plusMatcherReturn(val i: Int, val node: Node?)
+    fun plusMatcher(node: Node?, initialIndex: Int): plusMatcherReturn{
```

Comment:
```suggestion
    }

    data class plusMatcherReturn(val i: Int, val node: Node?)

    fun plusMatcher(node: Node?, initialIndex: Int): plusMatcherReturn{
```
Please run ktlint to make these small things get automatically corrected :)